diff -up shadow-1.4.1/extconf.rb~ shadow-1.4.1/extconf.rb
--- shadow-1.4.1/extconf.rb~	2012-04-20 10:48:06.724264584 -0400
+++ shadow-1.4.1/extconf.rb	2012-04-20 10:40:45.517013943 -0400
@@ -6,7 +6,8 @@
 require 'mkmf'
 require 'rbconfig'
 
-$CFLAGS = case RUBY_VERSION
+$CFLAGS = "#{$CFLAGS}"
+$CFLAGS += case RUBY_VERSION
           when /^1\.9/; '-DRUBY19'
           when /^2\./; '-DRUBY19'
           else; ''
